tests/test_loadReferenceObjects.py

```diff
@@ -121,8 +121,12 @@ def testFilterAliasMap(self):
                 # camera filters aliases are named <filter>_camFlux
                 if "camr" in filterMap:
                     self.assertEqual(getRefFluxField(refSchema, "camr"), "camr_camFlux")
+                # If defaultFilter is specified, use that instead if there's no mapping.
```

Comment:
Please clarify this comment. What is "that"? I suggest moving the comment and changing it to something along the lines of the following (assuming it is obvious that this test has no filterMap):
```
elif defaultFilter != "":
    # defaultFilter specified but we have no filterMap;
    # The chosen filter should be "camFlux"
    self.assertEqual(...)
else:
    # ... (please describe this case)
```